Userland/Libraries/LibCore/Process.cpp

```diff
@@ -12,13 +12,16 @@
 #include <AK/Vector.h>
 #include <LibCore/Process.h>
 #include <LibCore/System.h>
+#include <LibFileSystem/FileSystem.h>
```

Comment:
ah, one last thing. can you remove this include? it seems unused. And it's a library layering problem to reach into LibFileSystem from LibCore (LibFileSystem depends on LibCore)

Reply:
Whoops, this should have been `<LibCore/File.h>`. Fixed now.